diff -urBb rsync-2.6.5.org/rsyncd.conf.5 rsync-2.6.5/rsyncd.conf.5
--- rsync-2.6.5.org/rsyncd.conf.5	2005-07-23 16:01:33.135106000 +0200
+++ rsync-2.6.5/rsyncd.conf.5	2005-07-23 16:01:58.800205112 +0200
@@ -325,7 +325,7 @@
 you may find that passwords longer than 8 characters don\&'t work\&. 
 .IP 
 There is no default for the "secrets file" option, you must choose a name
-(such as \f(CW/etc/rsyncd\&.secrets\fP)\&.  The file must normally not be readable
+(such as \f(CW/etc/rsyncd/rsyncd\&.secrets\fP)\&.  The file must normally not be readable
 by "other"; see "strict modes"\&.
 .IP 
 .IP "\fBstrict modes\fP" 
@@ -623,13 +623,13 @@
         path = /data/cvs
         comment = CVS repository (requires authentication)
         auth users = tridge, susan
-        secrets file = /etc/rsyncd\&.secrets
+        secrets file = /etc/rsyncd/rsyncd\&.secrets
 
 .fi 
  
 
 .PP 
-The /etc/rsyncd\&.secrets file would look something like this:
+The /etc/rsyncd/rsyncd\&.secrets file would look something like this:
 .PP 
 .RS 
 \f(CWtridge:mypass\fP
@@ -640,7 +640,7 @@
 .PP 
 .SH "FILES" 
 .PP 
-/etc/rsyncd\&.conf or rsyncd\&.conf
+/etc/rsyncd/rsyncd\&.conf or rsyncd\&.conf
 .PP 
 .SH "SEE ALSO" 
 .PP 
diff -urBb rsync-2.6.5.org/rsyncd.conf.yo rsync-2.6.5/rsyncd.conf.yo
--- rsync-2.6.5.org/rsyncd.conf.yo	2005-07-23 16:01:33.136106000 +0200
+++ rsync-2.6.5/rsyncd.conf.yo	2005-07-23 16:01:58.800205112 +0200
@@ -69,7 +69,7 @@
 reread its config file.
 
 Note that you should bf(not) send the rsync daemon a HUP signal to force
-it to reread the tt(rsyncd.conf) file. The file is re-read on each client
+it to reread the tt(/etc/rsyncd/rsyncd.conf) file. The file is re-read on each client
 connection. 
 
 manpagesection(GLOBAL OPTIONS)
@@ -139,7 +139,7 @@
 
 dit(bf(path)) The "path" option specifies the directory in the daemon's
 filesystem to make available in this module.  You must specify this option
-for each module in tt(rsyncd.conf).
+for each module in tt(/etc/rsyncd/rsyncd.conf).
 
 dit(bf(use chroot)) If "use chroot" is true, the rsync daemon will chroot
 to the "path" before starting the file transfer with the client.  This has
@@ -292,7 +292,7 @@
 you may find that passwords longer than 8 characters don't work. 
 
 There is no default for the "secrets file" option, you must choose a name
-(such as tt(/etc/rsyncd.secrets)).  The file must normally not be readable
+(such as tt(/etc/rsyncd/rsyncd.secrets)).  The file must normally not be readable
 by "other"; see "strict modes".
 
 dit(bf(strict modes)) The "strict modes" option determines whether or not 
@@ -543,10 +543,10 @@
         path = /data/cvs
         comment = CVS repository (requires authentication)
         auth users = tridge, susan
-        secrets file = /etc/rsyncd.secrets
+        secrets file = /etc/rsyncd/rsyncd.secrets
 )
 
-The /etc/rsyncd.secrets file would look something like this:
+The /etc/rsyncd/rsyncd.secrets file would look something like this:
 
 quote(
 tt(tridge:mypass)nl()
diff -urBb rsync-2.6.5.org/rsync.h rsync-2.6.5/rsync.h
--- rsync-2.6.5.org/rsync.h	2005-05-03 19:00:47.000000000 +0200
+++ rsync-2.6.5/rsync.h	2005-07-23 16:01:58.801204960 +0200
@@ -30,7 +30,7 @@
 /* RSYNCD_SYSCONF is now set in config.h */
 #define RSYNCD_USERCONF "rsyncd.conf"
 
-#define DEFAULT_LOCK_FILE "/var/run/rsyncd.lock"
+#define DEFAULT_LOCK_FILE "/var/lock/rsyncd.lock"
 #define URL_PREFIX "rsync://"
 
 #define BACKUP_SUFFIX "~"
diff -urBb rsync-2.6.5.org/rsync.yo rsync-2.6.5/rsync.yo
--- rsync-2.6.5.org/rsync.yo	2005-07-23 16:01:33.135106000 +0200
+++ rsync-2.6.5/rsync.yo	2005-07-23 16:02:35.743588864 +0200
@@ -233,7 +233,7 @@
 
 An rsync daemon is configured using a configuration file.  Please see the
 rsyncd.conf(5) man page for more information.  By default the configuration
-file is called /etc/rsyncd.conf (unless the daemon is spawned via a remote
+file is called /etc/rsyncd/rsyncd.conf (unless the daemon is spawned via a remote
 shell--see below).
 
 manpagesection(RUNNING AN RSYNC DAEMON OVER A REMOTE SHELL PROGRAM)
@@ -1334,7 +1334,7 @@
 If standard input is a socket then rsync will assume that it is being
 run via inetd, otherwise it will detach from the current terminal and
 become a background daemon.  The daemon will read the config file
-(rsyncd.conf) on each connect made by a client and respond to
+(/etc/rsyncd/rsyncd.conf) on each connect made by a client and respond to
 requests accordingly.  See the rsyncd.conf(5) man page for more
 details.
 
@@ -1352,7 +1352,7 @@
 
 dit(bf(--config=FILE)) This specifies an alternate config file than
 the default.  This is only relevant when bf(--daemon) is specified.
-The default is /etc/rsyncd.conf unless the daemon is running over
+The default is /etc/rsyncd/rsyncd.conf unless the daemon is running over
 a remote shell program and the remote user is not root; in that case
 the default is rsyncd.conf in the current directory (typically $HOME).
 
@@ -2045,7 +2045,7 @@
 
 manpagefiles()
 
-/etc/rsyncd.conf or rsyncd.conf
+/etc/rsyncd/rsyncd.conf or rsyncd.conf
 
 manpageseealso()
 
diff -urBb rsync-2.6.5.org/support/rsyncstats rsync-2.6.5/support/rsyncstats
--- rsync-2.6.5.org/support/rsyncstats	2005-02-19 20:36:54.000000000 +0100
+++ rsync-2.6.5/support/rsyncstats	2005-07-23 16:01:58.801204960 +0200
@@ -12,7 +12,7 @@
 use Getopt::Long;
 
 # You may wish to edit the next line to customize for your default log file.
-$usage_file = "/var/adm/rsyncd.log";
+$usage_file = "/var/log/rsyncd.log";
 
 # Edit the following lines for default report settings.
 # Entries defined here will be over-ridden by the command line.
