diff -ur rsync-2.6.7.org/rsyncd.conf.5 rsync-2.6.7/rsyncd.conf.5
--- rsync-2.6.7.org/rsyncd.conf.5	2006-03-11 19:25:08.000000000 +0100
+++ rsync-2.6.7/rsyncd.conf.5	2006-03-11 21:37:33.616778500 +0100
@@ -344,7 +344,7 @@
 you may find that passwords longer than 8 characters don\&'t work\&. 
 .IP 
 There is no default for the "secrets file" option, you must choose a name
-(such as \f(CW/etc/rsyncd\&.secrets\fP)\&.  The file must normally not be readable
+(such as \f(CW/etc/rsyncd/rsyncd\&.secrets\fP)\&.  The file must normally not be readable
 by "other"; see "strict modes"\&.
 .IP 
 .IP "\fBstrict modes\fP" 
@@ -656,13 +656,13 @@
         path = /data/cvs
         comment = CVS repository (requires authentication)
         auth users = tridge, susan
-        secrets file = /etc/rsyncd\&.secrets
+        secrets file = /etc/rsyncd/rsyncd\&.secrets
 
 .fi 
  
 
 .PP 
-The /etc/rsyncd\&.secrets file would look something like this:
+The /etc/rsyncd/rsyncd\&.secrets file would look something like this:
 .PP 
 .RS 
 \f(CWtridge:mypass\fP
@@ -673,7 +673,7 @@
 .PP 
 .SH "FILES" 
 .PP 
-/etc/rsyncd\&.conf or rsyncd\&.conf
+/etc/rsyncd/rsyncd\&.conf or rsyncd\&.conf
 .PP 
 .SH "SEE ALSO" 
 .PP 
diff -ur rsync-2.6.7.org/rsyncd.conf.yo rsync-2.6.7/rsyncd.conf.yo
--- rsync-2.6.7.org/rsyncd.conf.yo	2006-03-11 19:25:04.000000000 +0100
+++ rsync-2.6.7/rsyncd.conf.yo	2006-03-11 21:37:33.616778500 +0100
@@ -66,7 +66,7 @@
 reread its config file.
 
 Note that you should bf(not) send the rsync daemon a HUP signal to force
-it to reread the tt(rsyncd.conf) file. The file is re-read on each client
+it to reread the tt(/etc/rsyncd/rsyncd.conf) file. The file is re-read on each client
 connection. 
 
 manpagesection(GLOBAL OPTIONS)
@@ -137,7 +137,7 @@
 
 dit(bf(path)) The "path" option specifies the directory in the daemon's
 filesystem to make available in this module.  You must specify this option
-for each module in tt(rsyncd.conf).
+for each module in tt(/etc/rsyncd/rsyncd.conf).
 
 dit(bf(use chroot)) If "use chroot" is true, the rsync daemon will chroot
 to the "path" before starting the file transfer with the client.  This has
@@ -309,7 +309,7 @@
 you may find that passwords longer than 8 characters don't work. 
 
 There is no default for the "secrets file" option, you must choose a name
-(such as tt(/etc/rsyncd.secrets)).  The file must normally not be readable
+(such as tt(/etc/rsyncd/rsyncd.secrets)).  The file must normally not be readable
 by "other"; see "strict modes".
 
 dit(bf(strict modes)) The "strict modes" option determines whether or not 
@@ -564,10 +564,10 @@
         path = /data/cvs
         comment = CVS repository (requires authentication)
         auth users = tridge, susan
-        secrets file = /etc/rsyncd.secrets
+        secrets file = /etc/rsyncd/rsyncd.secrets
 )
 
-The /etc/rsyncd.secrets file would look something like this:
+The /etc/rsyncd/rsyncd.secrets file would look something like this:
 
 quote(
 tt(tridge:mypass)nl()
diff -ur rsync-2.6.7.org/rsync.h rsync-2.6.7/rsync.h
--- rsync-2.6.7.org/rsync.h	2006-03-11 21:37:15.251630000 +0100
+++ rsync-2.6.7/rsync.h	2006-03-11 21:37:33.616778500 +0100
@@ -30,7 +30,7 @@
 /* RSYNCD_SYSCONF is now set in config.h */
 #define RSYNCD_USERCONF "rsyncd.conf"
 
-#define DEFAULT_LOCK_FILE "/var/run/rsyncd.lock"
+#define DEFAULT_LOCK_FILE "/var/lock/rsyncd.lock"
 #define URL_PREFIX "rsync://"
 
 #define BACKUP_SUFFIX "~"
diff -ur rsync-2.6.7.org/rsync.yo rsync-2.6.7/rsync.yo
--- rsync-2.6.7.org/rsync.yo	2006-03-11 21:37:15.267631000 +0100
+++ rsync-2.6.7/rsync.yo	2006-03-11 21:37:33.616778500 +0100
@@ -1749,7 +1749,7 @@
 If standard input is a socket then rsync will assume that it is being
 run via inetd, otherwise it will detach from the current terminal and
 become a background daemon.  The daemon will read the config file
-(rsyncd.conf) on each connect made by a client and respond to
+(/etc/rsyncd/rsyncd.conf) on each connect made by a client and respond to
 requests accordingly.  See the rsyncd.conf(5) man page for more
 details.
 
@@ -2477,7 +2477,7 @@
 
 manpagefiles()
 
-/etc/rsyncd.conf or rsyncd.conf
+/etc/rsyncd/rsyncd.conf or rsyncd.conf
 
 manpageseealso()
 
