diff -Naur rsync-2.5.6-orig/rsync.h rsync-2.5.6/rsync.h
--- rsync-2.5.6-orig/rsync.h	Sun Jan 26 21:11:16 2003
+++ rsync-2.5.6/rsync.h	Thu Jan 30 17:39:51 2003
@@ -26,10 +26,10 @@
 #define RSYNC_RSH_ENV "RSYNC_RSH"
 
 #define RSYNC_NAME "rsync"
-#define RSYNCD_SYSCONF "/etc/rsyncd.conf"
+#define RSYNCD_SYSCONF "/etc/rsyncd/rsyncd.conf"
 #define RSYNCD_USERCONF "rsyncd.conf"
 
-#define DEFAULT_LOCK_FILE "/var/run/rsyncd.lock"
+#define DEFAULT_LOCK_FILE "/var/lock/rsyncd.lock"
 #define URL_PREFIX "rsync://"
 
 #define BACKUP_SUFFIX "~"
diff -Naur rsync-2.5.6-orig/rsync.yo rsync-2.5.6/rsync.yo
--- rsync-2.5.6-orig/rsync.yo	Tue Jan 28 04:11:57 2003
+++ rsync-2.5.6/rsync.yo	Thu Jan 30 17:42:31 2003
@@ -214,7 +214,7 @@
 
 An rsync server is configured using a config file.  Please see the 
 rsyncd.conf(5) man page for more information.  By default the configuration
-file is called /etc/rsyncd.conf, unless rsync is running over a remote
+file is called /etc/rsyncd/rsyncd.conf, unless rsync is running over a remote
 shell program and is not running as root; in that case, the default name
 is rsyncd.conf in the current directory on the remote computer 
 (typically $HOME).
@@ -730,7 +730,7 @@
 If standard input is a socket then rsync will assume that it is being
 run via inetd, otherwise it will detach from the current terminal and
 become a background daemon.  The daemon will read the config file
-(rsyncd.conf) on each connect made by a client and respond to
+(/etc/rsyncd/rsyncd.conf) on each connect made by a client and respond to
 requests accordingly.  See the rsyncd.conf(5) man page for more
 details.
 
@@ -751,7 +751,7 @@
 
 dit(bf(--config=FILE)) This specifies an alternate config file than
 the default.  This is only relevant when --daemon is specified. 
-The default is /etc/rsyncd.conf unless the daemon is running over
+The default is /etc/rsyncd/rsyncd.conf unless the daemon is running over
 a remote shell program and the remote user is not root; in that case
 the default is rsyncd.conf in the current directory (typically $HOME).
 
@@ -1098,7 +1098,7 @@
 
 manpagefiles()
 
-/etc/rsyncd.conf or rsyncd.conf
+/etc/rsyncd/rsyncd.conf or rsyncd.conf
 
 manpageseealso()
 
diff -Naur rsync-2.5.6-orig/rsyncd.conf.5 rsync-2.5.6/rsyncd.conf.5
--- rsync-2.5.6-orig/rsyncd.conf.5	Mon Jan 27 04:07:18 2003
+++ rsync-2.5.6/rsyncd.conf.5	Thu Jan 30 17:46:11 2003
@@ -259,7 +259,7 @@
 you may find that passwords longer than 8 characters don\'t work\&. 
 .IP 
 There is no default for the "secrets file" option, you must choose a name
-(such as \f(CW/etc/rsyncd\&.secrets\fP)\&.  The file must normally not be readable
+(such as \f(CW/etc/rsyncd/rsyncd\&.secrets\fP)\&.  The file must normally not be readable
 by "other"; see "strict modes"\&.
 .IP 
 .IP "\fBstrict modes\fP" 
@@ -504,13 +504,13 @@
         path = /data/cvs
         comment = CVS repository (requires authentication)
         auth users = tridge, susan
-        secrets file = /etc/rsyncd\&.secrets
+        secrets file = /etc/rsyncd/rsyncd\&.secrets
 
 .fi 
  
 
 .PP 
-The /etc/rsyncd\&.secrets file would look something like this:
+The /etc/rsyncd/rsyncd\&.secrets file would look something like this:
 .PP 
 tridge:mypass 
 .br 
@@ -518,7 +518,7 @@
 .PP 
 .SH "FILES" 
 .PP 
-/etc/rsyncd\&.conf or rsyncd\&.conf
+/etc/rsyncd/rsyncd\&.conf or rsyncd\&.conf
 .PP 
 .SH "SEE ALSO" 
 .PP 
diff -Naur rsync-2.5.6-orig/rsyncd.conf.yo rsync-2.5.6/rsyncd.conf.yo
--- rsync-2.5.6-orig/rsyncd.conf.yo	Mon Jan 27 04:07:18 2003
+++ rsync-2.5.6/rsyncd.conf.yo	Thu Jan 30 17:48:52 2003
@@ -69,7 +69,7 @@
 reread its config file.
 
 Note that you should not send the rsync server a HUP signal to force
-it to reread the tt(rsyncd.conf) file. The file is re-read on each client
+it to reread the tt(/etc/rsyncd/rsyncd.conf) file. The file is re-read on each client
 connection. 
 
 manpagesection(GLOBAL OPTIONS)
@@ -128,7 +128,7 @@
 
 dit(bf(path)) The "path" option specifies the directory in the servers
 filesystem to make available in this module.  You must specify this option
-for each module in tt(rsyncd.conf).
+for each module in tt(/etc/rsyncd/rsyncd.conf).
 
 dit(bf(use chroot)) If "use chroot" is true, the rsync server will chroot
 to the "path" before starting the file transfer with the client.  This has
@@ -239,7 +239,7 @@
 you may find that passwords longer than 8 characters don't work. 
 
 There is no default for the "secrets file" option, you must choose a name
-(such as tt(/etc/rsyncd.secrets)).  The file must normally not be readable
+(such as tt(/etc/rsyncd/rsyncd.secrets)).  The file must normally not be readable
 by "other"; see "strict modes".
 
 dit(bf(strict modes)) The "strict modes" option determines whether or not 
@@ -437,17 +437,17 @@
         path = /data/cvs
         comment = CVS repository (requires authentication)
         auth users = tridge, susan
-        secrets file = /etc/rsyncd.secrets
+        secrets file = /etc/rsyncd/rsyncd.secrets
 )
 
-The /etc/rsyncd.secrets file would look something like this:
+The /etc/rsyncd/rsyncd.secrets file would look something like this:
 
 tridge:mypass nl()
 susan:herpass
 
 manpagefiles()
 
-/etc/rsyncd.conf or rsyncd.conf
+/etc/rsyncd/rsyncd.conf or rsyncd.conf
 
 manpageseealso()
 
diff -Naur rsync-2.5.6-orig/rsyncstats rsync-2.5.6/rsyncstats
--- rsync-2.5.6-orig/rsyncstats	Mon Nov 23 01:30:27 1998
+++ rsync-2.5.6/rsyncstats	Thu Jan 30 17:50:17 2003
@@ -23,7 +23,7 @@
 # ---------------------------------------------------------------------------
 
 # edit the next line to customize for your default log file
-$usage_file = "/var/adm/rsyncd.log";
+$usage_file = "/var/log/rsyncd";
 
 # Edit the following lines for default report settings.
 # Entries defined here will be over-ridden by the command line.
